docs/cugraph/source/api_docs/property_graph.rst

```diff
@@ -1,13 +1,29 @@
 ====================
 Property Graph
 ====================
-.. currentmodule:: cugraph
+.. currentmodule:: cugraph.structure.property_graph
```

Comment:
This is a limitation of the docs tool. I don't believe it can be changed.
I will write an issue to fix this when PropertyGraph is removed from experimental.